### DISCUSSION TITLE
Quesiton: does mux-player support 360 video?

### DISCUSSION THREAD

**Discussion by aminamos** [muxinc/elements#466] [Q&A]
Does the MUX Player support 360 degree video (i.e. can it do shape projection with controls for turning around and/or VR viewing support)? 

Example: https://github.com/videojs/videojs-vr

**Comment by aminamos** [comment]
At the moment, Mux Player cannot support 360 video controls.

**Reply by alexlinebrink** [reply]
Thanks! Are the plans to support in the future? Or any way to modify it to support those?
